### PR TITLE
add link to install guide

### PR DESCRIPTION
**Pull request by geoffcline**
**Is this a bug fix or adding new feature?**
revise docs

**What is this PR about? / Why do we need it?**
update guidance on EBS CSI IAM Policy, link to AWS docs (where appropriate). 

**What testing is done?** 
n/a

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ## Installation
 #### Set up driver permission
-The driver requires IAM permission to talk to Amazon EBS to manage the volume on user's behalf. [The example policy here](./example-iam-policy.json) defines these permissions. 
+The driver requires IAM permission to talk to Amazon EBS to manage the volume on user's behalf. [The example policy here](./example-iam-policy.json) defines these permissions. AWS maintains a managed policy, avilable at ARN `arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy`.
```
Comment (torredil) [inline docs/install.md:13]: nit: available.

Comment (geoffcline) [reply docs/install.md:13]: tyty!

**Comment by torredil** [comment]
@geoffcline Thanks, changes lgtm 👍
Will approve and unhold once the commits have been squashed.

**Comment by geoffcline** [comment]
@torredil done, thank you!!

**Review by torredil — APPROVED** [review]
/unhold

**Pull request merged**